test/dataclasses/test_answer.py

```diff
@@ -132,26 +132,67 @@ def test_protocol(self):
         )
         assert isinstance(answer, Answer)
 
-    def test_to_dict(self):
```

Comment:
I suggest we keep the plain `test_to_dict` and `test_from_dict` tests that don't have any `meta` in the `GeneratedAnswer` too.